govtool/frontend/src/components/organisms/DashboardCards/DirectVoterDashboardCard.tsx

```diff
@@ -98,7 +98,7 @@ export const DirectVoterDashboardCard = ({
             dataTestId: "learn-more-button",
             onClick: () =>
               openInNewTab(
-                "https://docs.gov.tools/how-to-use-the-govtool/using-govtool/dreps/retire-as-a-drep",
+                "https://docs.gov.tools/using-govtool/govtool-functions/dreps/register-as-a-drep",
               ),
           },
```

Comment:
Wouldn't https://docs.gov.tools/using-govtool/govtool-functions/direct-voting be a better suited link?